### PR TITLE
Rename deployment to fstool

### PR DESCRIPTION
**Pull request by tiltec**
Also rename database to fstool

### CODE REVIEW

File: deploy.md [hunk 1]
```diff
@@ -48,9 +48,10 @@ module = config.wsgi:application
 master = true
 processes = 2
 
-socket = /tmp/yunity-dev.sock
+socket = /tmp/fstool.sock
+http-socket = /tmp/fstool.http.sock
```
Comment (nicksellen) [inline deploy.md]: I think I used that so I could run curl commands against it locally on the server to check it responds ok. The non-http socket needs to be requested with uwsgi protocol and command line clients are less/not available.

File: deploy/remote.sh [hunk 2]
```diff
@@ -3,42 +3,43 @@
 set -e
 
 BRANCH=$1
+backend_dir=foodsaving-backend
```
Comment (nicksellen) [inline deploy/remote.sh]: Maybe this is `project_name` and then it can be reused in more places? (ready for the next name change ;))

**Comment by nicksellen** [comment]
How about renaming `fstool` -> `foodsaving`? (then foodsaving.yunity.org...). Otherwise we have two names fstool and foodsaving, and they are essentially the same thing?

File: deploy.md (hunk 1 above)
Comment (tiltec) [inline deploy.md]: yeah, found this in bash-history:
`echo -e "GET / HTTP/1.1\r\n" | sudo socat unix-connect:/tmp/yunity-dev.http.sock STDIO`

I guess I can remove it again from the deploy.md

**Comment by tiltec** [comment]
Hopefully we will get the domain foodsaving.org soon (November/December-ish) to host the tool. 

These yunity.org subdomains are just temporarily and for now, it's only a tool and does not contain foodsaving information (how to start your community etc) that I would expect on something called foodsaving.org

So, *no* to rename fstool -> foodsaving right now

File: deploy/remote.sh (hunk 2 above)
Comment (tiltec) [inline deploy/remote.sh]: I didn't see more places right now, and generally I'm overwhelmed with too generic things ;)

**Pull request merged**